### PR TITLE
Keep shutter open during Andor kinetic acquisition

### PR DESCRIPTION
**Pull request by eoinell**

### CODE REVIEW

File: nplab/instrument/camera/Andor/andor_sdk.py [hunk 1]
```diff
@@ -476,11 +477,11 @@ def capture(self):
             int         number of images taken
             tuple       shape of the images taken
         """
-
+        self.Shutter = (1, self.keep_shutter_open, 1, 1)
```
Comment (YagoDel) [inline nplab/instrument/camera/Andor/andor_sdk.py]: Tbf, I don't fully understand what each of the elements in the self.Andor.Shutter property do, so I don't know why we'd be changing property values every time we take an acquisition

Comment (eoinell) [reply nplab/instrument/camera/Andor/andor_sdk.py]: When in kinetic mode, if Shutter == (1,0,1,1) the shutter opens and closes between each spectrum which introduces vibrations, cuts off some of the signal and limits the time resolution of the measurement. When Shutter == (1,1,1,1) it's kept open. It's a handy option to have, Bart has been coding in this option manually before and after each capture.

File: nplab/instrument/camera/Andor/__init__.py [hunk 2]
```diff
@@ -218,8 +218,13 @@ def temperature_gui(self):
         self.temperature_display_thread.start()
     def update_temperature_display(self, temperature):
         self.temperature_lcdNumber.display(float(temperature))
+    
     def get_temperature(self):
         return self.Andor.CurrentTemperature
+    
+    def update_shutter_mode(self):
+        self.Andor.keep_shutter_open = self.keep_shutter_open_checkBox.isChecked()
```
[the remark this answers is not in the document]
Comment (eoinell) [reply nplab/instrument/camera/Andor/__init__.py]: The issue is that the this mode needs be turned on and off before and after the capture as it just makes the andor's shutter stay open all the time which should be avoided.

File: nplab/instrument/camera/Andor/andor_sdk.py (hunk 1 above)
Comment (YagoDel) [reply nplab/instrument/camera/Andor/andor_sdk.py]: Fair enough, for some reason I thought the SDK could do that automatically. 
So then I think that it should be done in the Andor class not the AndorBase class (it's a very specific use case). And (perhaps more importantly), this functionality shouldn't be setting the rest of the Shutter parameters.

Each of the values in the tuple mean something (type, mode, closing time, opening time), and they should be preserved

**Pull request merged**